resources/migrations/000_migrations.yaml

```diff
@@ -4521,659 +4574,852 @@ databaseChangeLog:
 # The Quartz Task Scheduler can use a DB to 'cluster' tasks and make sure they are only ran by a single instance where
 # using a multi-instance Metabase setup.
 
+# Quartz identifiers are upper-case in MySQL and H2 but lower-case in PostgreSQL for reasons... so we'll have to
+# define properties for EVERYTHING and use the correct identifiers
```

Comment:
Previously, we used to only use the "quote all identifiers" strategy for MySQL. Now we use it for all databases. The Quartz migrations were all defined with uppercase identifiers; for MySQL (since they were quoted) and for H2 (since it treats unquoted identifiers as uppercase) the Quartz tables and columns got uppercase identifiers; for Postgres since it treats unquoted identifiers as lowercase they got lowercase identifiers.

Now that we're quoting **everything** I had to create properties for all the quartz identifiers so we can match the old behavior. Don't worry, I generated these programmatically, so even tho it's like 600 LoC it didn't take long to do.